openqabot/commenter.py

```diff
@@ -2,6 +2,7 @@
 # SPDX-License-Identifier: MIT
 from __future__ import annotations
 
+from argparse import Namespace
```

Comment:
thought: I think this will not cause any issue on my current work on https://github.com/openSUSE/qem-bot/pull/322 but I'll rebase anyway 👍